packages/components/src/responsive-toolbar-group/index.tsx

```diff
@@ -36,11 +37,16 @@ const ResponsiveToolbarGroup = ( {
 	 * Rendering mode
 	 */
 	forceSwipe?: boolean;
+
+	/**
+	 * When false completely disables swipe at all breakpoints.
+	 */
+	swipeEnabled?: boolean;
 } ) => {
 	const classes = classnames( 'responsive-toolbar-group', className );
-	const shouldSwipe = useBreakpoint( swipeBreakpoint ) || forceSwipe;
+	const shouldSwipe = useBreakpoint( swipeBreakpoint );
 
-	if ( shouldSwipe ) {
+	if ( forceSwipe || ( swipeEnabled && shouldSwipe ) ) {
```

Comment:
I feel this'd be clearer if all checks are done in `shouldSwipe` since there are more conditions where we _should swipe_:

```js
const shouldSwipe = forceSwipe || ( swipeEnabled && useBreakpoint( swipeBreakpoint ) );

if ( shouldSwipe ) {
```

The alternative would be to rename `shouldSwipe` to something else:

```js
const isWithinSwipeBreakpoint = useBreakpoint( swipeBreakpoint );

if ( forceSwipe || ( swipeEnabled && isWithinSwipeBreakpoint ) ) {
```

Reply:
Changed the name of the constant.